superset/config.py

```diff
@@ -797,6 +797,8 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 
 # Any config options to be passed as-is to the webdriver
 WEBDRIVER_CONFIGURATION: Dict[Any, Any] = {}
+# The webdriver to use supports "chrome" and "firefox"
+WEBDRIVER_TYPE = "firefox"
```

Comment:
let's use EMAIL_REPORTS_WEBDRIVER or just converge those 2
easy solution would be:
```
EMAIL_REPORTS_WEBDRIVER = EMAIL_REPORTS_WEBDRIVER
```

Reply:
Maybe took it too far, but I actually removed it now. Tell me if it's ok by you